appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5119,6 +5123,39 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @Description("")
   String MaximumRangeMethods();
 
+  @DefaultMessage(
+    "<table border='1' cellpadding='8' cellspacing='0'>" +
+    "<thead>" +
+    "<tr>" +
+    "<th>Action</th>" +
+    "<th>Key Combination</th>" +
+    "</tr>" +
+    "</thead>" +
+    "<tbody>" +
+    "<tr><td>Focus Component search box</td><td>/</td></tr>" +
+    "<tr><td>Focus Components tree</td><td>T</td></tr>" +
+    "<tr><td>Focus Viewer</td><td>V</td></tr>" +
+    "<tr><td>Focus Properties Panel</td><td>P</td></tr>" +
+    "<tr><td>Switch between Designer and Block editor</td><td>Ctrl + Alt</td></tr>" +
+    "<tr><td>Rename Component</td><td>Alt + N</td></tr>" +
+    "<tr><td>Reset Connection</td><td>Alt + Shift + R</td></tr>" +
+    "<tr><td>Refresh Companion Screen</td><td>Alt + R</td></tr>" +
+    "<tr><td>Open/Close Backpack</td><td>Alt + B</td></tr>" +
```

Comment:
The Alt + B shortcut was implemented in a way so that it works only when the user clicks on the blockly workspace to make it active, this was done because if the shortcut was pressed in the designer view then the backpack flyout opens but in a weird way where the blocks are outside of the flyout.